third_party/2/concurrent/futures/__init__.pyi

```diff
@@ -13,11 +16,13 @@ class Future(Generic[_T]):
     def done(self) -> bool: ...
     def result(self, timeout: float = ...) -> _T: ...
     def exception(self, timeout: float = ...) -> Any: ...
+    def exception_info(self, timeout: float = ...) -> Tuple[Any, Optional[TracebackType]]: ...
```

Comment:
timeout should be `Optional[float]`. Looks like that's also true for `result` and `exception` above; feel free to change those too.